src/CreateEntityMappingView/CreateEntityMappingView.js

```diff
@@ -215,7 +242,43 @@ class CreateEntityMappingView extends ConnectionComponent {
 		this.setState({mappings});
 	}
 
-	loadAttributes() {
+	// is called to load the existing entity mapping in the edit view
+	entityMappingPending() {
```

Comment:
naming is not very good => why pending? isn´t it used to load data into the view?
A name like loadDataIntoEditView or something like that would be sufficient.